files/etc/profile.d/workarea.sh

```diff
@@ -62,18 +66,17 @@ do not put in your protocol like https://, this is handled for you):"
   set -a
   . /etc/workarea.env
   set +a
-  sed -i "s/www.shop.com/$host/" /srv/shop/config/initializers/workarea.rb
 
   if [[ "$host" != "$ip" ]]; then
     echo "Setting up SSL for $host with certbot..."
     certbot --nginx
   else
     echo "Generating self-signed SSL cert..."
     pushd /etc/ssl/private
-    openssl genrsa -out workarea.key 2048
-    openssl rsa -in workarea.key -out workarea.key
-    openssl req -sha256 -new -key workarea.key -out workarea.csr -subj "/CN=$ip"
-    openssl x509 -req -sha256 -days 365 -in workarea.csr -signkey workarea.key -out workarea.crt
+    openssl genrsa -out workarea.key 2048 > /dev/null 2>&1
```

Comment:
The folks from DO complained that they could see a weird RNG error thrown by OpenSSL. I couldn't figure out a way to get rid of it, and anyway the cert/key gets generated properly, so I'm just removing all output since it's just a distraction anyway.

Reply:
It seems relatively risky to swallow all the output here. Why not fix the actual error instead of sweeping it under the rug?

Reply:
Because it's not a real error in the sense that it doesn't affect the generation of a self-signed cert. Feels like a waste of time to dive deep into OpenSSL to fix the error if it doesn't actually affect the outcome. Additionally, the self-signed cert is generated for evaluation purposes, it doesn't actually have to be "secure" and the feedback loop on this project is so long that it really doesn't feel like I should spend a day or so trying to work this OpenSSL "error" out.

Reply:
Well, it's sweeping everything under the rug. So what happens if something else goes wrong in the future?